src/redemptions/redemption.service.ts

```diff
@@ -3,12 +3,19 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 import { Injectable, NotFoundException } from '@nestjs/common';
 import { DecisionStatus, KycStatus, Redemption, User } from '@prisma/client';
+import { instanceToPlain } from 'class-transformer';
 import { ApiConfigService } from '../api-config/api-config.service';
 import { JumioTransactionRetrieveResponse } from '../jumio-api/interfaces/jumio-transaction-retrieve';
+import { IdDetails } from '../jumio-kyc/kyc.service';
 import { PrismaService } from '../prisma/prisma.service';
 import { BasePrismaClient } from '../prisma/types/base-prisma-client';
 import { UserPointsService } from '../user-points/user-points.service';
 
+export type CalculatedStatus = {
```

Comment:
this is only used in one place, can you inline it unless we use it in more than one place?